test/test_mps.py

```diff
@@ -6328,7 +6328,7 @@ def helper(shape, alpha=1.0, memory_format=torch.contiguous_format):
                 elu_result.backward(gradient=grad)
                 elu_result_cpu.backward(gradient=cpu_grad)
 
-                self.assertEqual(elu_result, elu_result_cpu)
+                self.assertEqual(elu_result, elu_result_cpu, atol=1e-5, rtol=1e-5)
```

Comment:
It should be possible to fix this increased error, since the MPSGraph impl had lower error. I'll look into it